### PR TITLE
refactor(dedicated.server): move secondary dns module

### PR DESCRIPTION
**Pull request by ghyenne**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/multi-az` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-12286 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
Refactor secondary dns section, move secondary dns module

### CODE REVIEW

**Review by darsene — APPROVED** [review]

File: packages/manager/modules/bm-server-components/src/secondary-dns/add/secondary-dns-add.controller.js
Comment (anooparveti) [inline packages/manager/modules/bm-server-components/src/secondary-dns/add/secondary-dns-add.controller.js:1]: **nitpick:** Remove `$stateParams` and add it as an input to the component.

File: packages/manager/modules/bm-server-components/src/secondary-dns/delete/secondary-dns-delete.controller.js
Comment (anooparveti) [inline packages/manager/modules/bm-server-components/src/secondary-dns/delete/secondary-dns-delete.controller.js:1]: **nitpick:** Remove `$stateParams` and add it as an input to the component.

File: packages/manager/modules/bm-server-components/src/secondary-dns/secondary-dns.controller.js
Comment (anooparveti) [inline packages/manager/modules/bm-server-components/src/secondary-dns/secondary-dns.controller.js:1]: **nitpick:** Remove `$stateParams` and add it as an input to the component.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/bm-server-components/src/secondary-dns/add/secondary-dns-add.controller.js
Comment (ghyenne) [reply packages/manager/modules/bm-server-components/src/secondary-dns/add/secondary-dns-add.controller.js:1]: I removed it. I reuse this.server.name.

File: packages/manager/modules/bm-server-components/src/secondary-dns/secondary-dns.controller.js
Comment (ghyenne) [reply packages/manager/modules/bm-server-components/src/secondary-dns/secondary-dns.controller.js:1]: I removed it. I reuse this.server.name.

File: packages/manager/modules/bm-server-components/src/secondary-dns/delete/secondary-dns-delete.controller.js
Comment (ghyenne) [reply packages/manager/modules/bm-server-components/src/secondary-dns/delete/secondary-dns-delete.controller.js:1]: I removed it. I reuse this.server.name and $transition$.params for this.domain

**Review by anooparveti — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Pull request merged**